### ISSUE TITLE
Where is routing defined?

### ISSUE THREAD

**Issue by lfjeff** [gimler/symfony-rest-edition#19]
Were is the routing configured for each command?

I can't find a routing.yml file or anything else.  The routing seems to be done magically and I'm not even sure where to search.

**Comment by willdurand** [comment]
No magic here, but it uses the FOSRestBundle's custom route loader:
- https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/Resources/doc/5-automatic-route-generation_single-restful-controller.md
- https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/Resources/doc/6-automatic-route-generation_multiple-restful-controllers.md

**Comment by lfjeff** [comment]
Thanks, I had already looked at those documents, but I still don't fully understand everything and did not make connection.  It would probably make sense if I had been using the REST bundle for several months, but I just installed it yesterday and much of what it does still seems like magic.  I've managed to get things working by using @Route annotations, but the automatic route creation is still somewhat baffling.

A few explicit examples or comments would be helpful for beginners.  According to various Symfony REST examples I've found, there seem to be about a thousand ways to configure everything.  When you're first starting a project, it's sometimes hard to determine the pros and cons of each method.
